pkg/types/nutanix/defaults/platform.go

```diff
@@ -6,8 +6,5 @@ import (
 
 // SetPlatformDefaults sets the defaults for the platform.
 func SetPlatformDefaults(p *nutanix.Platform) {
-	if p.PrismAPICallTimeout == nil {
-		timeout := nutanix.DefaultPrismAPICallTimeout
-		p.PrismAPICallTimeout = &timeout
-	}
+
```

Comment:
I don't think this should be removed, instead do the similar in case of baremetals, for example,  https://github.com/openshift/installer/blob/ca4ca4145fab5d53a10282f0cac76b74502ca9b0/pkg/asset/agent/installconfig.go#L389-L391 else it will affect IPI flow

Cc @sadasu @zaneb

Reply:
Actually, setting this default value is not necessary. Because the default value is already set anyway internally, see https://github.com/yanhua121/openshift-installer/blob/master/pkg/types/nutanix/helpers.go#L44. Now it caused some issue with agent install use case since this field is not supported by the agent-installer. So I removed it. I tested the IPI, it works as expected.